src/app/shell/personal-cabinet/provider/create-provider/create-photo-form/create-photo-form.component.html

```diff
@@ -11,8 +11,11 @@
       </mat-option>
     </mat-select>
   </mat-form-field>
-
-  <label class="step-label"> Опис <span class="step-required">*</span></label>
+  <div class="step-label-combined">
+    <label class="step-label"> Опис <span class="step-required">*</span></label>
+    <label class="step-characters-count">
+    {{PhotoFormGroup.get('description').value.length}}/{{constants.MAX_DESCRIPTION_LENGTH}}</label>
```

Comment:
please save description control into the variable and use instead to avoid calling .get() method every second:
eg. {{description.value.length}}